src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuPaymentRequestDocumentPreRules.java

```diff
@@ -86,4 +90,25 @@ protected void clearWireTransferValues(PaymentRequestWireTransfer preqWireTransf
 	        preqWireTransfer.setPreqPayeeAccountName(null);
 	    }
 
+    @SuppressWarnings("deprecation")
+    protected boolean validateCheckStubLength(PaymentRequestDocument document) {
```

Comment:
There are a couple very similar validateCheckStubLength() functions, would it be helpful to extract some of this logic to avoid duplication?

Reply:
At the time I was reluctant about pulling the check stub question/validation logic out of the pre-rules classes. However, given the number of documents using this logic, I'll see if I can find a shared spot for it, possibly within the `CuCheckStubService`.